src/scripts/news/NewsProvider.js

```diff
@@ -0,0 +1,49 @@
+let articles = []
```

Comment:
I like the calling these objects articles. Should we rename this file to ArticleProvider?